global.json

```diff
@@ -1,5 +1,5 @@
 {
     "msbuild-sdks": {
-        "Microsoft.Quantum.Sdk": "0.24.213686-alpha"
+        "Microsoft.Quantum.Sdk": "0.24.214451-alpha"
```

Comment:
There is likely something that I'm not aware of, but based on what I know I assume that the refs to the _alpha_ versions are a temporary measure, and the refs to the _alpha_ versions will not actually get merged to `main` (but instead the refs to alpha will be replaced with the refs to non-alpha before merging to main).

Reply:
Yes, they are.